### ISSUE TITLE
Words specified as italic in title (Latex command) only appear as squares

### ISSUE THREAD

**Issue by ladyteam** [JabRef/jabref#8522]
### JabRef version

5.5 (latest release)

### Operating system

GNU / Linux

### Details on version and operating system

Opensuse Tumbleweed

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

The simple entry example with italic used in the title:
```
@Article{Parlinski2021,
  author    = {Krzysztof Parlinski and Przemys{\l}aw Piekarz},
  journal   = {Journal of Raman Spectroscopy},
  title     = {\textit{Ab initio} determination of {R}aman spectra},
  year      = {2021},
  number    = {7},
  pages     = {1346--1359},
  volume    = {52},
  doi       = {10.1002/jrs.6129},
}

```
Will reproduce first symbols as squares. Related to issue 2784 this was fixed in Windows. What's about Linux then? I've been trying to change the font in configuration file since there is no possibility to do it in gui, but nothing changes.

Thank you in advance,
Eugene

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
Yeah, and I still encounter this under mac as well. 
<img width="1156" alt="grafik" src="https://user-images.githubusercontent.com/320228/155197864-483bcd83-a585-4a1a-ba98-bcc9e3b74669.png">

@k3KAW8Pnf7mkmdSMPHz27  Did you ever find a workaround/solution for this? 
Refs #2784

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Nope, I didn't look at it further. Back in https://github.com/JabRef/jabref/issues/3356#issuecomment-635531500 it was quite easy to trigger by resizing the window.

**Comment by Siedlerchr** [comment]
Hm, okay, I played a it around with it, and NFC + NFD both do render squares.  The other two spit out normal text. 
I noticed that with css I can style the whole content in italic. So it can be displayed in theory.

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Perhaps it is still the fonts? JavaFX seems to have had a lot of issues with them, and I can't replicate this issue in Ubuntu 20.4.

**Comment by ThiloteE** [comment]
what happens if you change the command to ` title     = {\\textit{Ab initio} determination of {R}aman spectra},` ?

**Comment by ladyteam** [comment]
@ThiloteE nothing happens. But we have some news. I have Desktop with Opensuse 15.4 fresh install and Laptop with Opensuse 15.4 upgrade from 15.3(upgraded from 15.2 and so on). Jabref version 5.0 installed on both computers and the one installed in Laptop reproduce the problem while in Desktop the issue is somehow "resolved". Have no idea what is wrong with Laptop Opensuse. What should I check?
Info:
java installed in Laptop: java-1_8_0-openjdk-1.8.0.332-150000.3.67.1.x86_64
java installed in Desktop: java-11-openjdk-11.0.16.0-150000.3.83.1.x86_64

**Comment by ladyteam** [comment]
After upgrading java in Laptop (java-11-openjdk-11.0.16.0-150000.3.83.1.x86_64) still the problem persist :(

**Comment by bwakkie** [comment]
I just compiled from the git branch today and it is working but only in the Custom preview style. Not in the 5 other available styles like IEEE etc...

**Comment by Siedlerchr** [comment]
JabRef dev version JabRef 6.0-PullRequest13937.953--2025-10-10--140d65e 

Main table is rendered correctly on mac os  15.6.1

<img width="1296" height="406" alt="Image" src="https://github.com/user-attachments/assets/750d36cd-764d-41d7-8ed1-efd65ae1060e" /> 

Customized Entry style shows it correctly, CSL style fails
